app/src/hooks/api/useAxios.test.ts

```diff
@@ -13,7 +13,7 @@ describe('APIError', () => {
       }
     } as Partial<AxiosError>;
 
-    const apiError = new APIError(error as unknown as AxiosError);
+    const apiError = new APIError((error as unknown) as AxiosError);
```

Comment:
Looks like the same formatting thing I mentioned here: https://github.com/bcgov/biohubbc-platform/pull/46/files?file-filters%5B%5D=.tsx&show-deleted-files=true&show-viewed-files=true#r890750038